apps/hash-api/src/cache/redis.ts

```diff
@@ -18,14 +22,15 @@ export class RedisCache extends DataSource implements CacheAdapter {
 
   constructor(logger: Logger, cfg: RedisConfig) {
     super();
-    this.client = new AsyncRedisClient(logger, cfg);
-    this.get = this.client.get;
-    this.set = this.client.set;
-    this.setExpiring = this.client.setex;
-    this.rpush = this.client.rpush;
+    this.client = setupRedisClient(logger, cfg);
+    this.get = (key) => this.client.get(key);
+    this.set = (key, value) => this.client.set(key, value).then();
```

Comment:
Shouldn't be affected and we should still get a thrown error
![image](https://user-images.githubusercontent.com/6988668/234515981-52747705-91c4-403b-bca2-4b9a546b68af.png)

Reply:
Right sorry, `then` returns another `Promise` (I was thinking about how it generally takes a callback so it might continue executing)

With that in mind, what's the purpose of the `.then()` calls here?

Reply:
The class defines the operation to return `Promise<void>;` and the library returns `Promise<"OK">. It's to void the output

Reply:
yes 656e1a7a0cdf507ac99cf834eaeced3e6ba42436